### PR TITLE
[WIP] Expand Software Offering: TLP Plugin for Laptops

### PR DESCRIPTION
**Pull request by 89luca89**
Add TLP/Powertop tweak
This is important to Laptop/Mobile users to improve battery life

This PR is part of a rework on another PR
I splitted in many little PR as requested

### CODE REVIEW

File: plugins/tlp.plugin/install.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin.bash
+
+dnf -y install tlp tlp-rdw powertop smartmontools
+systemctl enable powertop.service
+systemctl enable tlp.service
+systemctl enable tlp-sleep.service
+tlp start true
+semanage fcontext -a -t ifconfig_var_run /run/tlp/lock_tlp
```
Comment (kwizart) [inline plugins/tlp.plugin/install.sh]: If this service cannot work without changing the SElinux context, please report a bug and add a comment to the bug here. If ever the SElinux context need to be changed in the next tlp release, you will create a bug by using an wrong context here.

File: plugins/tlp.plugin/install.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin.bash
```
Comment (kwizart) [inline plugins/tlp.plugin/install.sh]: typo

File: plugins/tlp.plugin/uninstall.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin.bash
```
Comment (kwizart) [inline plugins/tlp.plugin/uninstall.sh]: again, #!/bin/bash

File: plugins/tlp.plugin/install.sh (hunk 1 above)
Comment (89luca89) [reply plugins/tlp.plugin/install.sh]: This fixes only a warning
Without it, TLP Works correclty but `Selinux Troubleshooter` will fire a notification warning each sleep/resume

I can remove the context change for now?

Comment (kwizart) [reply plugins/tlp.plugin/install.sh]: You can remove it, but submitting a bugzilla to the tlp component is needed, applications aren't expected to fire selinux warning.

Comment (Abhinav1217) [reply plugins/tlp.plugin/install.sh]: Is there any recommended Selinux policy regarding tlp and powertop services?

**Comment by 89luca89** [comment]
@kwizart  
If you are interested I have searched what specifically TLP/Powertop does and developed a udev rule

https://gist.github.com/89luca89/29be32da857fa4aa59b09ab626929d2f

This does 90% of what TLP does (and covers 100% of the powertop functionality)
Only missing things from TLP are the thinkpad specific functions (charge/discharge treshold) and CPU customization ([Here](https://gist.github.com/89luca89/b44731223311e3c4130d216c24dc2574) a script that can go with the udev rule switching AC/Battery)

Just with this rule the battery life of laptops can be improved dramatically

**Pull request closed without merging**